nilearn/glm/first_level/experimental_paradigm.py

```diff
@@ -185,7 +179,8 @@ def _check_unexpected_columns(events):
 STRATEGY = {"modulation": "sum"}
 
 
-def _handle_duplicate_events(events):
+def sum_modulation_of_duplicate_events(events):
```

Comment:
(Very minor) Slight inconsistency is naming: we leave strategy as a global variable, but call the corresponding function "sum"

Reply:
to be fair I am not even sure why those globals were used: I guess to allow for other strategies in the future ?

Reply:
Will revert to the older name though I find it less specific regarding what the function actually does.

Reply:
done